devicon.json

```diff
@@ -1544,7 +1544,15 @@
         "color": "#a9bacd",
         "aliases": [
             {
-                "base": "plain",
+                "base": "original",
+                "alias": "original-wordmark"
+            },
+            {
+                "base": "original",
+                "alias": "plain"
+            },
+            {
+                "base": "original",
                 "alias": "plain-wordmark"
```

Comment:
Looks like the `C` icon seems missing, or having multiple versions ?
![image](https://github.com/user-attachments/assets/06ebfa4c-df45-49c3-a911-8c2211734244)